test/game_statistics_test.rb

```diff
@@ -53,4 +53,12 @@ def test_percentage_home_wins
     assert_equal 40.00, @stat_tracker.percentage_home_wins
   end
 
+  def test_total_score_sum
+    assert_equal 45, @stat_tracker.total_score_sum
```

Comment:
Maybe it doesn't feel like we need this helper now, but I think it will come in handy later.